Servers/UI/OJS.Servers.Ui/ClientApp/src/common/constants.ts

```diff
@@ -15,9 +15,17 @@ enum ContestResultType {
     Full = 'full',
 }
 
+enum ContestValidationErrors {
+    NotFound = 'Contest not found!',
+    Expired = 'Contest expired!',
+    NotRegistered = 'You are not registered for this contest!',
+    NotEligible = 'You can not take part in the contest!',
+}
```

Comment:
Messages should be in a dictionary. Enum can be used for keys, i.e.

`ContestErrorMessages[ContestErrors.NotFound]`

Reply:
I used the built in Record type with enum for a key and string for value.